src/app/fdctl/run/tiles/fd_verify.c

```diff
@@ -95,33 +95,57 @@ after_frag( void *             _ctx,
 
   fd_verify_ctx_t * ctx = (fd_verify_ctx_t *)_ctx;
 
+  /* XXX is this check needed? */
```

Comment:
It doesn't really matter. We are guaranteed the txn starts before the high watermark, so there is at least FD_TPU_DCACHE_MTU bytes available to read/write. If the txn is too short or there's garbage, it will fail to verify.